videoConf/views.py

```
@@ -10,16 +10,17 @@
 def meeting(request: HttpRequest) -> HttpResponse:
-    context = { 
-        'room_name': f"{settings.JITSI_ROOM_PREFIX}{request.user.name}_{int(time.time())}",
-        'user_name': request.user.username,
+    context = {
+        # 'room_name': f"{settings.JITSI_ROOM_PREFIX}{request.user.name}_{int(time.time())}",
+        'room_name': "sadi_jifat",
+        'display_name': request.user.username,
         'JITSI_MAGIC_COOKIE': settings.JITSI_MAGIC_COOKIE,
         'SRI_HASH': settings.SRI_HASH,
-        }
+        'jwt': settings.JWT
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)